pulp.spec

```diff
@@ -9,6 +9,7 @@
 %define pulp_admin 1
 %define pulp_client_oauth 1
 %define pulp_server 1
+%define pulp_streamer 1
```

Comment:
Does this need to be set to 0 in the above ``if`` block? Or is there something that automatically will do that?